src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -24,6 +24,9 @@ public partial class DllNotFoundException : System.TypeLoadException
         public DllNotFoundException(string message) { }
         public DllNotFoundException(string message, System.Exception inner) { }
     }
+    public partial struct RuntimeArgumentHandle
```

Comment:
Yes ArgIterator has been removed from nestandard (https://github.com/dotnet/standard/commit/6e3ad137e2b4e6556ca9467b5e41e887554e944c). If there are more types in that bucket please file an issue on the standard repo calling them out.